.cirrus.yml

```diff
@@ -40,8 +40,8 @@ style_task:
   integration_test_script: >
     if ! cmake --build build --target integration; then
       tar -czf $(cat ARTIFACT_NAME).tar.gz -C build vast-integration-test
-      scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i upload_key \
-        $(cat ARTIFACT_NAME).tar.gz mule@tenzir.dfn-cert.de:integration/vast/
+      rsync --archive --compress --rsh "ssh -i upload_key -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" \
+        --progress $(cat ARTIFACT_NAME).tar.gz mule@tenzir.dfn-cert.de:/home/mule/integration/vast/
```

Comment:
```suggestion
        --progress "$(cat ARTIFACT_NAME).tar.gz" mule@tenzir.dfn-cert.de:/home/mule/integration/vast/
```

Reply:
@dominiklohmann thanks for pointing this out. We should then, to my mind, change all variable resolutions to use double quotes instead.